docs/package-lock.json

```diff
@@ -1,853 +1,8 @@
 {
```

Comment:
I'd probably not make any changes here.

Reply:
I didn't even touch this locally, not sure why it's in here

Reply:
I suppose your build updated dependencies.